gdal/configure.ac

```diff
@@ -2565,6 +2565,48 @@ AC_SUBST([HAVE_OPENJPEG],$HAVE_OPENJPEG)
 AC_SUBST([OPENJPEG_VERSION],$OPENJPEG_VERSION)
 
 dnl ---------------------------------------------------------------------------
+dnl Select a JPEG XR (jxrlib) Library to use, or disable driver.
+dnl ---------------------------------------------------------------------------
+
+AC_ARG_WITH(jpegxr,[  --with-jpegxr[=ARG]     Include JPEG XR support via jxrlib library (ARG=path)],,)
```

Comment:
Yes, it is. It is waiting for decision if internal `jxrlib` is accepted or not, then the script needs appropriate update  of course.